lib/Sema/TypeCheckDeclPrimary.cpp

```diff
@@ -2146,6 +2146,11 @@ class DeclChecker : public DeclVisitor<DeclChecker> {
     (void) VD->getPropertyWrapperAuxiliaryVariables();
     (void) VD->getPropertyWrapperInitializerInfo();
     (void) VD->getImplInfo();
+    // Declarations inside types are handled in checkConformancesInContext() to
+    // avoid cycles involving associated type inference.
+    if (!VD->getDeclContext()->isTypeContext()) {
+      (void)getActorIsolation(VD);
```

Comment:
I actually just realized that this invocation of getActorIsolation() is redundant since it is already called on all `ValueDecl` instances during `DeclChecker::visit()` (from which this method is called). About to push a commit removing this...